docs/optimization_guide/dldt_deployment_optimization_tput.md

```diff
@@ -37,14 +34,15 @@ Most generally, throughput-oriented inference applications should:
          :fragment: [compile_model]
 
    @endsphinxdirective
-   * Query the `ov::optimal_number_of_infer_requests` from the `ov::CompiledModel` (resulted from compilation of the model for a device) to create the number of the requests required to saturate the device
-* Use the Async API with callbacks, to avoid any dependency on the requests' completion order and possible device starvation, as explained in the [common-optimizations section](@ref openvino_docs_deployment_optimization_guide_common)
+   * Query the `ov::optimal_number_of_infer_requests` from the `ov::CompiledModel` (resulted from compilation of the model for a device) to create the number of the requests required to saturate the device.
+* Use the Async API with callbacks, to avoid any dependency on the requests' completion order and possible device starvation.  explained in the [common-optimizations section](@ref openvino_docs_deployment_optimization_guide_common).
```

Comment:
```suggestion
* Use the Async API with callbacks, to avoid any dependency on the completion order of the requests and possible device starvation, as explained in the [common-optimizations section](@ref openvino_docs_deployment_optimization_guide_common).
```